src/plugins/dam/components/uploader/Uploader.jsx

```diff
@@ -252,6 +265,57 @@ class Uploader extends React.Component {
   }
 }
 
+class SaveButton extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggleDisable = this.toggleDisable.bind(this);
+
+    this.state = {
+      isDisabled: true
+    }
+  }
+
+  toggleDisable(e) {
+    setTimeout(() => {
+      const { delegate } = this.props;
+      const hasCredits = !!document.querySelector('.select__single-value');
```

Comment:
@digitalartcrew dom querying in a react app is an anti pattern only to be used as a last resort - there must be another piece of state or prop that indicates the same thing

Reply:
Yes this was interesting because the was not an actual input or select element to capture value